src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -330,13 +330,12 @@ class HeaderHelpMenuUI extends Component<Props, State> {
         })}
         onClick={this.onMenuButtonClick}
       >
-        <EuiIcon type="questionInCircle" size="m" />
         <EuiIcon
-          type="help"
+          type="questionInCircle"
           size="m"
           title={intl.formatMessage({
             id: 'core.ui.chrome.headerGlobalNav.helpMenuButtonTitle',
-            defaultMessage: 'Help',
+            defaultMessage: 'Help menu',
```

Comment:
I think the finalized guidance is just "Help": https://github.com/opensearch-project/documentation-website/issues/3351#issuecomment-1466379362